src/wp-admin/options-general.php

```diff
@@ -25,14 +25,25 @@
 add_action( 'admin_head', 'options_general_add_js' );
 
 $options_help = '<p>' . __( 'The fields on this screen determine some of the basics of your site setup.' ) . '</p>' .
-	'<p>' . __( 'Most themes display the site title at the top of every page, in the title bar of the browser, and as the identifying name for syndicated feeds. The tagline is also displayed by many themes.' ) . '</p>';
+	'<p>' . __( 'Most themes show the site title at the top of every page, in the title bar of the browser, and as the identifying name for syndicated feeds. Many themes also show the tagline.' ) . '</p>';
 
 if ( ! is_multisite() ) {
-	$options_help .= '<p>' . __( 'The WordPress URL and the site URL can be the same (example.com) or different; for example, having the WordPress core files (example.com/wordpress) in a subdirectory instead of the root directory.' ) . '</p>' .
-		'<p>' . __( 'If you want site visitors to be able to register themselves, as opposed to by the site administrator, check the membership box. A default user role can be set for all new users, whether self-registered or registered by the site admin.' ) . '</p>';
+	$options_help .= '<p>' . __( 'Two terms you will want to know are the WordPress URL and the site URL. The WordPress URL is where the core WordPress installation files are, and the site URL is the address a visitor uses in the browser to go to your site.' ) . '</p>' .
+		'<p>' . sprintf(
+			/* translators: %s: Documentation URL. */
+			__( 'Though the terms refer to two different concepts, in practice, they can be the same address or different. For example, you can have the WordPress core files in the root directory (<code>https://example.com</code>), in which case the two URLs would be the same. Or the <a href="%s">WordPress files can be in a subdirectory</a> (<code>https://example.com/wordpress</code>). In that case, the site URL and the WordPress URL would be different.' ),
+			__( 'https://wordpress.org/documentation/article/giving-wordpress-its-own-directory/' )
+		) . '</p>' .
+		'<p>' . sprintf(
+			/* translators: 1: http://, 2: https:// */
+			__( 'Both site URLs and WordPress URLs can start with either %1$s or %2$s. A URL starting with %2$s will also show a padlock next to the address in the browser address bar. Both of those things signal to visitors that your site meets some basic security requirements, which can build trust with your users and with search engines.' ),
```

Comment:
```suggestion
			__( 'Both WordPress URLs and site URLs can start with either %1$s or %2$s. A URL starting with %2$s will also show a padlock next to the address in the browser address bar. Both of those things signal to visitors that your site meets some basic security requirements, which can build trust with your users and with search engines.' ),
```
Nitpick: "WordPress URL" comes before "site URL" in the previous paragraph.